doc/lpc18/background.tex

```diff
@@ -2,7 +2,8 @@ \section{Background concepts}\label{sec:background}
 
 \subsection{The P4 programming language}
 
-(This section is adapted from~\cite{budiu-osr17}.)  P4 is a relatively
+\mycomment{(This section is adapted from~\cite{budiu-osr17}.)}
```

Comment:
Maybe we should say in the introduction that we used these documents. Some paragraphs are almost entirely lifted.